src/main/java/com/gdschongik/gdsc/domain/study/application/MentorStudyDetailService.java

```diff
@@ -68,4 +69,19 @@ public void publishStudyAssignment(Long studyDetailId, AssignmentCreateRequest r
 
         log.info("[StudyMentorService] 과제 개설 완료: studyDetailId={}", studyDetailId);
     }
+
+    @Transactional
+    public void updateStudyAssignment(Long studyDetailId, AssignmentUpdateRequest request) {
+        Member currentMember = memberUtil.getCurrentMember();
+        StudyDetail studyDetail = studyDetailRepository
+                .findById(studyDetailId)
+                .orElseThrow(() -> new CustomException(STUDY_DETAIL_NOT_FOUND));
+
+        studyDetailValidator.validateUpdateStudyAssignment(currentMember, studyDetail, request);
+
+        studyDetail.updateAssignment(request.title(), request.deadLine(), request.descriptionNotionLink());
+        studyDetailRepository.save(studyDetail);
+
+        log.info("[StudyMentorService] 과제 수정 완료: studyDetailId={}", studyDetailId);
```

Comment:
```suggestion
        log.info("[MemtorStudyDetailService] 과제 수정 완료: studyDetailId={}", studyDetailId);
```

제가 class 이름 바꾸면서 로그에 있는 걸 깜빡했네요...
위에 있는 다른 로그들도 같이 수정 부탁드립니다.